bin/phpunit

```diff
@@ -8,7 +8,7 @@ if (!ini_get('date.timezone')) {
 if (is_file(dirname(__DIR__).'/vendor/phpunit/phpunit/phpunit')) {
     define('PHPUNIT_COMPOSER_INSTALL', dirname(__DIR__).'/vendor/autoload.php');
     require PHPUNIT_COMPOSER_INSTALL;
-    PHPUnit\TextUI\Command::main();
+    exit((new \PHPUnit\TextUI\Application())->run($GLOBALS['argv']));
```

Comment:
This is needed when you upgrade to v10 phpunit. After all `PHPUnit\TextUI\Command` does not exists anymore.